services-camel/src/main/java/edu/unc/lib/cdr/fulltext/FulltextRouter.java

```diff
@@ -32,36 +29,28 @@
  *
  */
 public class FulltextRouter extends RouteBuilder {
-	@BeanInject(value = "binaryMetadataProcessor")
-	private BinaryMetadataProcessor mdProcessor;
 	
 	@BeanInject(value = "fulltextProcessor")
 	private FulltextProcessor ftProcessor;
 	
 	
 	public void configure() throws Exception {
-		String mimetypePattern = "^(text/|application/pdf|application/msword|application/vnd\\.|application/rtf|application/powerpoint|application/postscript).*$";
+		String mimetypePattern = "^(text/|application/pdf|application/msword"
```

Comment:
What do you think about putting this in a class constant instead to improve visibility/maintainability?